2020-06-29_13-41-47: DF Exception: Sub: R1147P, Sess: 1, FileNotFoundError, [Errno 2] No such file or directory: '/scratch/john/SWR_scratch/SWR_FR1_R1147P_1_HPC_surrounding_recall_soz_in.p', file: <ipython-input-7-6c198b13def7>, line no: 38
2020-06-29_13-41-50: DF Exception: Sub: R1243T, Sess: 3, FileNotFoundError, [Errno 2] No such file or directory: '/scratch/john/SWR_scratch/SWR_FR1_R1243T_3_HPC_surrounding_recall_soz_in.p', file: <ipython-input-7-6c198b13def7>, line no: 38
2020-06-29_13-41-53: DF Exception: Sub: R1334T, Sess: 2, FileNotFoundError, [Errno 2] No such file or directory: '/scratch/john/SWR_scratch/SWR_FR1_R1334T_2_HPC_surrounding_recall_soz_in.p', file: <ipython-input-7-6c198b13def7>, line no: 38
2020-06-29_13-41-53: DF Exception: Sub: R1342M, Sess: 1, FileNotFoundError, [Errno 2] No such file or directory: '/scratch/john/SWR_scratch/SWR_FR1_R1342M_1_HPC_surrounding_recall_soz_in.p', file: <ipython-input-7-6c198b13def7>, line no: 38
